api/src/controllers/programs.ts

```diff
@@ -99,6 +117,13 @@ const programsRouter = router({
     if (rowsToInsert.length) await db.insert(plannerMajor).values(rowsToInsert);
   }),
   /** @todo add `setPlannerMinor` (or similarly named) operation for updating a minor */
+  saveSelectedMinor: publicProcedure.input(MinorProgramSchema).mutation(async ({ input }) => {
+    const { plannerId, minorIds } = input;
+    await db.delete(plannerMinor).where(eq(plannerMinor.plannerId, plannerId));
+
+    const rowsToInsert = minorIds.map((minorId) => ({ plannerId, minorId }));
+    if (rowsToInsert.length) await db.insert(plannerMinor).values(rowsToInsert);
```

Comment:
This looks like it'd work, but for multiple, we should add/remove individual rows instead of overriding every row every time. Since it's also the case for majors, let's file a new issue for this.